examples/type-check-expression/type_check_expression.description

```diff
@@ -1,2 +1,2 @@
-// Type check expression can be used to assert the runtime type of a value in Ballerina.
+// Type check expression can be used to assert the runtime type of a value or any expression in Ballerina.
```

Comment:
The previous statement sound right to me. It always checks the type of a value at runtime.

Being able to give any expression during compile time is different to that IMO.

Reply:
Revert to previous statement.